ROADMAP.md

```diff
@@ -0,0 +1,9 @@
+# Lima roadmap
+
+Instead of using a static text file, Lima uses the `roadmap` label on Github issues to designate features or bug fixes that we plan to implement.
```

Comment:
> roadmap label

Can we add a hyperlink to https://github.com/lima-vm/lima/issues?q=is%3Aissue+is%3Aopen+label%3Aroadmap ?